cms-lite/src/main/resources/webapp/partials/widgets/stream-resource-modal.html

```diff
@@ -41,7 +41,7 @@ <h4 class="modal-title" id="streamResourceLabel">{{msg('cmslite.header.resource'
 
             <div class="modal-footer">
                 <a ui-if="mode == 'read'"  class="btn btn-default pull-left"
-                        href="../cmsliteapi/stream/{{select.language}}/{{select.name}}/"
+                        href="{{ '../cmsliteapi/resource/stream/' + select.language + '/' + select.name }}"
```

Comment:
This button should lead us directly to the resource content. I believe after changing this link to contain resource, it will display us the String/Stream Content entity associated with this content. Can you verify and fix if necessary?